privacyidea/static/templates/header.html

```diff
@@ -7,19 +7,19 @@
     <meta name="description" content="">
     <meta name="author" content="Cornelius Kölbel" >
     <link rel="icon" type="image/png" href="{{ instance }}/static/favicon.png">
+    <link href="{{ instance }}/static/contrib/css/bootstrap.css" rel="stylesheet">
```

Comment:
OK, I see. You moved this to the top, so that other stylesheets may overwrite the original bootstrap css. +1